### PR TITLE
Proposal: add option to use wp-env for dev

### PR DESCRIPTION
**Pull request by vcanales**
**Steps to test**:
Follow current installation instructions, and then the instructions added [here](https://github.com/amnestywebsite/humanity-theme/pull/342/files#diff-1bc238721e899c47c88c121abe8b2c87a92971b5c0c7929b99f9226d160e0ac3R24).

### CODE REVIEW

File: private/package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "watch": "webpack --mode development --watch",
     "lint:scripts": "eslint ./src/",
     "lint:styles": "stylelint \"src/**/*.scss\"",
-    "lint": "yarn lint:scripts; yarn lint:styles"
+    "lint": "yarn lint:scripts; yarn lint:styles",
+    "env": "npx @wordpress/env"
```
Comment (matiasbenedetto) [inline private/package.json:19]: Given that the project uses `yarn` instead of `npm`, should`npx` be replaced by the equivalent `yarn dlx` ?

Comment (vcanales) [reply private/package.json]: Replaced in 96ff301c4260bfae47ba0f507c53ea4b7bab88d7

File: docs/contributors/code.md [hunk 2]
```diff
@@ -18,7 +19,17 @@ This guide assumes you have an available Virtual Host or Docker environment thro
   This step installs our [PHPCS](https://github.com/PHPCSStandards/PHP_CodeSniffer) toolchain, and will allow you to scan your code changes for any stylistic incompatibilities. It's important to run PHPCS (`composer lint`) prior to creating any Pull Requests, as PRs are auto-rejected if any issues are found.  
 - Install and build the theme assets: `cd ./private && yarn && yarn build`  
   This project requires node v20+; you may need to enable corepack first: `corepack enable`.  
-- Download any Required or Recommended Plugins into the `./wp-content/plugins` directory. Follow plugin instructions for installation steps.  
+- Download any Required or Recommended Plugins into the `./wp-content/plugins` directory, or add them to `.wp-env.json` if using `@wordpress/env` as a development environment. Follow plugin instructions for installation steps.
+
+#### Local development environment with `@wordpress/env`
+If you choose to, you can use `@wordpress/env` which will provision a local development environment using Docker. 
+To get started run `yarn env start`.
```
Comment (jaymcp) [inline docs/contributors/code.md:26]:
```suggestion
To get started run `yarn env start` from within the `./private` directory.
```

**Review by jaymcp — COMMENTED** [review]
I like this change, thanks! Feel free to finalise it when you are ready 🙂

**Review by jaymcp — APPROVED** [review]
Works a treat, thanks!

Side note: I'm not sure if it's currently possible with WP env, but it'd be great if we could tell it to activate the theme on environment start.

**Comment by vcanales** [comment]
> Side note: I'm not sure if it's currently possible with WP env, but it'd be great if we could tell it to activate the theme on environment start.

Good point! Followed up here: https://github.com/amnestywebsite/humanity-theme/pull/348

**Comment by creativecoder** [comment]
Seems like we might as well add all of the required plugins to the wp-env configuration?

PR for that here: https://github.com/amnestywebsite/humanity-theme/pull/349

**Pull request merged**